src/Widgets/Footer.vala

```diff
@@ -34,7 +34,11 @@ namespace Network {
             this.pack_end (airplane_switch);
 
             airplane_switch.notify["active"].connect (() => {
-                client.networking_set_enabled (!client.networking_get_enabled ());
+                if (airplane_switch.active) {
```

Comment:
Can we replace this whole if statement by: `client.networking_set_enabled (!airplane.switch.active)`?